cookbooks/aws-parallelcluster-common/test/recipes/controls/setup_envars_spec.rb

```diff
@@ -0,0 +1,22 @@
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+control 'path_contains_required_directories' do
+  title 'System path contains required directories'
+
+  describe file('/etc/profile.d/path.sh') do
+    it { should exist }
+    its('mode') { should cmp '0755' }
+    its('owner') { should eq 'root' }
+    its('group') { should eq 'root' }
+    its('content') { should match 'PATH_REQUIRED_DIRECTORIES=\(/usr/local/sbin /usr/local/bin /sbin /bin /usr/sbin /usr/bin /opt/aws/bin\)' }
```

Comment:
question: here we check the path in the file, but I guess we should check also the env PATH variable, am I wrong?

Reply:
Unfortunately, the variable is not set at this point.
However, I already have another change with a workaround to test it, another PR will follow.